dists/base/base-d7-modules.make

```diff
@@ -227,13 +260,13 @@ libraries[backbone][type] = "library"
 
 ; Underscore.js (navbar)
 libraries[underscore][download][type] = "get"
-libraries[underscore][download][url] = "https://github.com/jashkenas/underscore/archive/1.5.2.zip"
+libraries[underscore][download][url] = "https://github.com/documentcloud/underscore/archive/master.zip"
```

Comment:
Good point. You're right. Master could lead to problems, when they introduce big changes at some point.

I just decided for one of the make files (here it was RamsaltBase)...